server/runtime/src/main/server/server/conf/infinispan-local.xml

```diff
@@ -39,6 +39,10 @@
       </security>
 
       <endpoints socket-binding="default" security-realm="default">
+         <ip-filter>
+            <reject from="172.16.0.0/16"/>
+            <accept from="127.0.0.0/8"/>
+         </ip-filter>
```

Comment:
The default `infinispan.xml` does not have named connectors. Should we update this to:
```
         <hotrod-connector name="hotrod"/>
         <rest-connector name="rest"/>
``` 
This then allows users to disable either connector via the CLI without any additional config.

Reply:
If you don't specify a name, one is generated: https://github.com/infinispan/infinispan/blob/master/server/runtime/src/main/java/org/infinispan/server/configuration/hotrod/HotRodServerConfigurationParser.java#L105

Reply:
I didn't realise. Is this documented anywhere so the user is aware of this?

Reply:
I see it is nvm